packit_service/worker/build/babysit.py

```diff
@@ -142,7 +143,16 @@ def update_copr_builds(build_id: int, builds: Iterable["CoprBuildModel"]) -> boo
         bool: Whether the run was successful, False signals the need to retry.
     """
     copr_client = CoprClient.create_from_config_file()
-    build_copr = copr_client.build_proxy.get(build_id)
+    try:
+        build_copr = copr_client.build_proxy.get(build_id)
+    except copr.v3.CoprNoResultException:
+        logger.error(
```

Comment:
please change this to info

logger.error lines end up in our sentry and this is definitely not an error which needs to be acted upon - as you say, builds are gone, nothing we can do about it

thanks!

Reply:
Thanks for pointing this out, good to know :) I will push the fix once https://github.com/packit/packit-service/pull/1268 merges so that I don't have to rebase twice...